### ISSUE TITLE
rake doc:guides not working

### ISSUE THREAD

**Issue by zzak** [rails/rails#18028]
```
~/code/bench/ruby-bench/src/FourTwoApp => be rake --trace doc:guides
** Invoke doc:guides (first_time)
** Execute doc:guides
rake aborted!
NoMethodError: undefined method `gem_dir' for nil:NilClass
/Users/zzak/.gem/ruby/2.2.0/gems/railties-4.2.0.rc3/lib/rails/tasks/documentation.rake:66:in `block (2 levels) in <top (required)>'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:240:in `call'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:240:in `block in execute'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:235:in `each'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:235:in `execute'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:179:in `block in invoke_with_call_chain'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:172:in `invoke_with_call_chain'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/task.rb:165:in `invoke'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:150:in `invoke_task'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:106:in `block (2 levels) in top_level'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:106:in `each'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:106:in `block in top_level'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:115:in `run_with_threads'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:100:in `top_level'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:78:in `block in run'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'
/Users/zzak/.rubies/ruby-trunk/lib/ruby/2.2.0/rake/application.rb:75:in `run'
/Users/zzak/.rubies/ruby-trunk/bin/rake:33:in `<main>'
Tasks: TOP => doc:guides
```

**Comment by justincase** [comment]
Currently not experiencing any errors with:

    $ ruby -v
    ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin14]
    
    $ gem -v
    2.4.5

    $ bundle -v
    Bundler version 1.7.11

    $ rails -v
    Rails 4.2.0

**Comment by zzak** [comment]
This should be closed as of cd7cc5254b090ccbb84dcee4408a5acede25ef2a
